lua/luasnip/nodes/choiceNode.lua

```diff
@@ -81,18 +78,18 @@ end
 ---
 ---@param choices (LuaSnip.Node|LuaSnip.Node[])[] A list of nodes that can be
 ---switched between. If a list of nodes is passed as a choice, it will be turned
----into a snippetNode.  
+---into a snippetNode.
```

Comment:
Oh, this paragraph-break was intentional, it'll be preserved in the generated `DOC.md`, and causes a linebreak when viewed on github